presto-main/src/main/java/io/prestosql/snapshot/SingleInputSnapshotState.java

```diff
@@ -122,7 +124,9 @@ else if (state.get() == TaskSnapshotManager.NO_STATE) {
                     LOG.error("BUG! State of component %s has never been stored successfully before snapshot %d", restorableId, snapshotId);
                 }
                 else {
+                    Stopwatch timer = Stopwatch.createStarted();
                     restorable.restore(state.get(), pagesSerde);
+                    timer.stop();
```

Comment:
need to capture spill time also.. move timer.stop() line 143 (after isSpilled() if block)

Reply:
Time for Loading spilled file is captured in common place when ever load file is called inside loadSpilledFiles(), as no deserialization involved for spilled files, this part is excluded. Do we still need to modify?